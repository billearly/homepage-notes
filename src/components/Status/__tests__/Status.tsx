import React from "react";
import renderer from "react-test-renderer";
import { Status } from "../";

describe("Status", () => {
  test("renders correctly", () => {
    const component = (
      <Status percentage={50} />
    );

    const elem = renderer
      .create(component)
      .toJSON();

    expect(elem).toMatchSnapshot();
  });
});