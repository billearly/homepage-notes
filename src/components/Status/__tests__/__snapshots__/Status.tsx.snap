// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Status renders correctly 1`] = `
<div
  className="status"
>
  <div
    className="status__bar"
    style={
      Object {
        "width": "50%",
      }
    }
  />
</div>
`;
